Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -1,198 +1,228 @@
-.. include:: /Includes.rst.txt
-.. index::
-   Site handling; Basics
-   Module; Site management
-.. _sitehandling-basics:
+..  include:: /Includes.rst.txt
+..  index::
+    Site handling; Basics
+    Module; Site management
+..  _sitehandling-basics:
 
 ======
 Basics
 ======
 
-TYPO3 site handling and configuration is the starting point for creating new web sites.
-The corresponding modules are found in the TYPO3 backend in the section :guilabel:`Site management`.
+TYPO3 site handling and configuration is the starting point for creating new
+websites. The corresponding modules are found in the TYPO3 backend in the
+section :guilabel:`Site management`.
 
 A site configuration consists of the following parts:
 
-* Base URL configurations: the domain(s) to access my site:
-* :ref:`Language configuration<sitehandling-addingLanguages>`: the languages of my site.
-* :ref:`Error Handling<sitehandling-errorHandling>`: error behavior of my site (For example: configuration of custom 404 pages)
-* :ref:`Static Routes<sitehandling-staticRoutes>`: static routes of my site (For example :file:`robots.txt` on a per site base)
-* Routing Configuration: How shall routing behave for this site
+*   Base URL configurations: the domain(s) to access my site.
```

Comment:
This might be easier to read as a definition list?

Reply:
It's okay for me:
![image](https://user-images.githubusercontent.com/2566282/201485422-41e9bc3b-64e5-4c69-8e98-9e790ed8111c.png)